cli/main.rs

```diff
@@ -340,16 +340,22 @@ pub(crate) static UNSTABLE_GRANULAR_FLAGS: &[(
   ),
 ];
 
-pub(crate) fn unstable_exit_cb(_feature: &str, api_name: &str) {
-  // TODO(bartlomieju): change to "The `--unstable-{feature}` flag must be provided.".
-  eprintln!("Unstable API '{api_name}'. The --unstable flag must be provided.");
+pub(crate) fn unstable_exit_cb(feature: &str, api_name: &str) {
+  eprintln!(
+    "Unstable API '{api_name}'. The `--unstable-{}` flag must be provided.",
+    feature
+  );
   std::process::exit(70);
 }
 
-#[allow(dead_code)]
-pub(crate) fn unstable_warn_cb(feature: &str) {
+// TODO(bartlomieju): remove when `--unstable` flag is removed.
+pub(crate) fn unstable_warn_cb(feature: &str, api_name: &str) {
   eprintln!(
-    "The `--unstable` flag is deprecated, use --unstable-{feature} instead."
+    "⚠️  {}",
+    colors::yellow(format!(
+      "The `{}` API was used with `--unstable` flag. The `--unstable` flag is deprecated, use granular `--unstable-{}` instead.\nLearn more at: https://docs.deno.com/runtime/manual/tools/unstable_flags",
```

Comment:
Perhaps we should add a note that it'll be removed in Deno v2.

Reply:
Yes, that makes sense. @littledivy can do update the message here.

Also the message should say that this flag has no effect when used with `deno check` subcommand.

Reply:
Please remove `Use --unsatble* instead`. We no longer conditionally type check - unstable APIs are now always type-checked.